Gemfile.lock

```diff
@@ -7,41 +7,73 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
-    diff-lcs (1.1.3)
-    ffi (1.1.5)
-    guard (1.4.0)
-      listen (>= 0.4.2)
-      thor (>= 0.14.6)
-    guard-rspec (2.1.0)
-      guard (>= 1.1)
-      rspec (~> 2.11)
-    listen (0.5.3)
-    mock_redis (0.5.4)
-    rake (0.9.2.2)
-    rb-fsevent (0.9.2)
-    rb-inotify (0.8.8)
-      ffi (>= 0.5.0)
-    redis (3.2.0)
-    rspec (2.11.0)
-      rspec-core (~> 2.11.0)
-      rspec-expectations (~> 2.11.0)
-      rspec-mocks (~> 2.11.0)
-    rspec-core (2.11.1)
-    rspec-expectations (2.11.3)
-      diff-lcs (~> 1.1.3)
-    rspec-mocks (2.11.3)
-    thor (0.16.0)
-    timecop (0.5.2)
+    coderay (1.1.3)
+    diff-lcs (1.5.0)
+    ffi (1.15.5)
+    formatador (1.1.0)
+    guard (2.18.0)
+      formatador (>= 0.2.4)
+      listen (>= 2.7, < 4.0)
+      lumberjack (>= 1.0.12, < 2.0)
+      nenv (~> 0.1)
+      notiffany (~> 0.0)
+      pry (>= 0.13.0)
+      shellany (~> 0.0)
+      thor (>= 0.18.1)
+    guard-compat (1.2.1)
+    guard-rspec (4.7.3)
+      guard (~> 2.1)
+      guard-compat (~> 1.1)
+      rspec (>= 2.99.0, < 4.0)
+    listen (3.7.1)
+      rb-fsevent (~> 0.10, >= 0.10.3)
+      rb-inotify (~> 0.9, >= 0.9.10)
+    lumberjack (1.2.8)
+    method_source (1.0.0)
+    mock_redis (0.31.0)
+      ruby2_keywords
+    nenv (0.3.0)
+    notiffany (0.1.3)
+      nenv (~> 0.1)
+      shellany (~> 0.0)
+    pry (0.14.1)
+      coderay (~> 1.1)
+      method_source (~> 1.0)
+    rake (13.0.6)
+    rb-fsevent (0.11.1)
+    rb-inotify (0.10.1)
+      ffi (~> 1.0)
+    redis (4.6.0)
+    rspec (3.11.0)
+      rspec-core (~> 3.11.0)
+      rspec-expectations (~> 3.11.0)
+      rspec-mocks (~> 3.11.0)
+    rspec-core (3.11.0)
+      rspec-support (~> 3.11.0)
+    rspec-expectations (3.11.0)
```

Comment:
### [rspec-expectations](https://github.com/rspec/rspec-expectations)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 2.11.3 → 3.11.0

[_[change-log](https://github.com/rspec/rspec-expectations/blob/v3.11.0/Changelog.md), [source-code](https://github.com/rspec/rspec-expectations), [gem-diff](https://my.diffend.io/gems/rspec-expectations/2.11.3/3.11.0)_]





<details>
<summary>Commits</summary>

A change of **1969** commits. See the full changes on [the compare page](https://github.com/rspec/rspec-expectations/compare/v2.11.3...v3.11.0).

These are the first 10 commits:
- (76ed5cf) [Bump version to 3.9.0.pre.](https://github.com/rspec/rspec-expectations/commit/76ed5cff28168a2eaf43a313850cd132b1141e6f)
- (3a0b5e4) [call to_hash only when object is not a Hash in RSpec::Matche…](https://github.com/rspec/rspec-expectations/commit/3a0b5e49776825425235b6ce8eabb99a03486c21)
- (31f108e) [Forwardport 3.8.1 release notes.](https://github.com/rspec/rspec-expectations/commit/31f108eae68e31ee65f0622c9ef4a488a1602943)
- (2c19f38) [Support checking initialize when checking respond_to(:new)](https://github.com/rspec/rspec-expectations/commit/2c19f383c941e4a2a8988cee14a88775fe99425b)
- (e20fcac) [Merge pull request #1072 from rspec/respond_to_checks_initia…](https://github.com/rspec/rspec-expectations/commit/e20fcacf887189630c8666bda2077f9ccd480a94)
- (8947285) [Update changelog for #1072](https://github.com/rspec/rspec-expectations/commit/89472853437828e6460cd4bc69708df7ff81e0c5)
- (06fca6b) [Updated travis build scripts (from rspec-dev)](https://github.com/rspec/rspec-expectations/commit/06fca6ba32513a83d0f4ab44a13f0c2d643b296f)
- (81b6039) [Prevent Github vulnerability warning](https://github.com/rspec/rspec-expectations/commit/81b60399f624ac42e5e49d73afa687471757f07a)
- (4e8af84) [Merge pull request #1073 from jkowens/3-8-maintenance](https://github.com/rspec/rspec-expectations/commit/4e8af84e5a78cd0c774df0e13eeb8c6ec08c1104)
- (66a9fa4) [Changelog for #1073](https://github.com/rspec/rspec-expectations/commit/66a9fa4d5471a4ea17da593833a567fbecda6e8e)


</details>